app/uk/gov/hmrc/exports/migrations/changelogs/notification/MakeParsedDetailsOptional.scala

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.exports.migrations.changelogs.notification
+
+import java.util
+
+import com.mongodb.client.MongoDatabase
+import org.bson.Document
+import org.mongodb.scala.bson.conversions.Bson
+import org.mongodb.scala.model.Filters.{and, exists, or, eq => feq}
+import org.mongodb.scala.model.UpdateOneModel
+import org.mongodb.scala.model.Updates.{combine, set, unset}
+import play.api.Logger
+import uk.gov.hmrc.exports.migrations.changelogs.{MigrationDefinition, MigrationInformation}
+
+import scala.collection.JavaConverters._
+
+class MakeParsedDetailsOptional extends MigrationDefinition {
+
+  private val logger = Logger(this.getClass)
+
+  private val INDEX_ID = "_id"
+  private val MRN = "mrn"
+  private val DATE_TIME_ISSUED = "dateTimeIssued"
+  private val STATUS = "status"
+  private val ERRORS = "errors"
+  private val DETAILS = "details"
+
+  private val collectionName = "notifications"
+
+  override val migrationInformation: MigrationInformation =
+    MigrationInformation(
+      id = "CEDS-2800 Make the parsed detail fields mrn, dateTimeIssued, status & error optional",
+      order = 9,
```

Comment:
I'm wondering if it couldn't be iterated from 1. Mongock history is separate, and it seems no migration has been executed with new tool yet.

Reply:
Ok set to '1'